setup.py

```diff
@@ -37,7 +37,7 @@
     'futures >= 3.2.0; python_version < "3.2"',
 ]
 extras = {
-    "grpc": "grpcio >= 1.29.0, < 2.0dev",
+    "grpc": "grpcio >= 1.33.2, < 2.0dev",
```

Comment:
Did the `grpico` library really add a feature (the non-experimental `aio` module) in a patch release?  That seems odd.

Reply:
Nah, it doesn't add new features, just adding a stable namespace, the existing "experimental" namespace still works. This is more or less a clean-up.